nbbuild/antsrc/org/netbeans/nbbuild/ResolveList.java

```diff
@@ -81,4 +102,52 @@ public void execute () throws BuildException {
         
         getProject().setNewProperty(name,value);
     }        
+
+    private void seekHarderExecute() {
```

Comment:
This would be hard to write in Ant using existing tasks, thus, if `path` and `dir` is set, I compute the values here in Java.